### ISSUE TITLE
column description bug

### ISSUE THREAD

**Issue by reza1615** [man-group/dtale#386]
In the column description, these metric have bugs

- **Rows w/ Hidden char**: count Arabic charters as hidden char for example count تست as hidden which is not hidden
- **Rows w/ accent**: count as 
    - Num
    - Accent
    - Hidden
- **Rows w/ number**: doesn't count number form example if a row is foo123 it doesn't count it as num

![image](https://user-images.githubusercontent.com/2202483/103562431-12555380-4e89-11eb-9801-56bcc52acc5a.png)

**Test**: on http://alphatechadmin.pythonanywhere.com/dtale/main/4
- In the **str_val** modify cells as 
    - 123
    - تست
    - foo123
    - fooÀ
    - [ZWNJ](https://en.wikipedia.org/wiki/Zero-width_non-joiner)

The correct column description should be
   - numeric chars: 2
   - accent chars: 1
   - hidden char: 1
   
I checked the regexes in the https://github.com/man-group/dtale/issues/335 and it was ok
https://regex101.com/r/GPxA1F/1
https://regex101.com/r/GPxA1F/2

**Comment by aschonfeld** [comment]
So the issue with numeric chars was just a front-end mapping issue.  Accent charts seems to be fine using the data you gave me:
![image](https://user-images.githubusercontent.com/11547371/103606128-b2d86180-4ee3-11eb-8188-66e3e061427d.png)

This is the regex I'm using to compute hidden characters:
```
from string import printable
df['var1'].str.count(r'[^{}]+'.format(printable)`).astype(bool).sum()
```

Looks like it is saying that `تست`, `fooÀ` & `ZWNJ` contain hidden characters.  Is there something wrong with `string.printable`?

**Comment by reza1615** [comment]
printable just check ASCII table so it is perfect for English language and doesn't support accent characters and Unicode characters
- for numeric chars please use `[\d]+` it supports all languages numbers like ۱۲۳123
- for hidden chars please use this regex
```python
printable =r'\w \!\"#\$%&\'\(\)\*\+,\-\./:;<»«؛،ـ\=>\?@\[\\\]\^_\`\{\|\}~'
df['var1'].str.count(r'[^{}]+'.format(printable)`).astype(bool).sum()

```
test the regex: https://regex101.com/r/cF5AVW/4

**Comment by reza1615** [comment]
Also please update the Cleaning function (remove hidden characters)
![image](https://user-images.githubusercontent.com/2202483/103700644-c67ed900-4f72-11eb-97af-d5e5e68e819b.png)

**Comment by reza1615** [comment]
And cleaning function to remove numbers

**Comment by aschonfeld** [comment]
fixed in v1.31.0
